lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnJobSubmissionHandler.java

```diff
@@ -69,7 +72,8 @@
       final ClasspathProvider classpath,
       final ConfigurationSerializer configurationSerializer,
       final JobUploader uploader,
-      @Parameter(JVMHeapSlack.class) final double jvmSlack) throws IOException {
+      @Parameter(JVMHeapSlack.class) final double jvmSlack,
+      @Parameter(JobQueue.class) String defaultQueueName) throws IOException {
```

Comment:
The defaultQueueName does not need to be given by user right? It will be automatically figured out from the cluster?